index.html

```diff
@@ -59,5 +59,217 @@ <h3 class="connect">LET’S CONNECT</h3>
+          </p>
+          <ul class="tech-stack">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <button class="btn-primary-outline">See project</button>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="image">
+          <img src="images/work2.png" alt="second project" />
+        </div>
+
+        <div class="card-content">
+          <h3 class="card-title"></h3>
+          <ul class="project-experience">
+            <li class="first-list">CANOPY</li>
+            <li class="other-list"><span></span>Full Stack Dev</li>
+            <li class="other-list"><span></span>2015</li>
+          </ul>
+          <p class="project-summary">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="tech-stack">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <button class="btn-primary-outline">See project</button>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="image">
+          <img src="images/work3.png" alt="third project" />
+        </div>
+
+        <div class="card-content">
+          <h3 class="card-title"></h3>
+          <ul class="project-experience">
+            <li class="first-list">CANOPY</li>
+            <li class="other-list"><span></span>Full Stack Dev</li>
+            <li class="other-list"><span></span>2015</li>
+          </ul>
+          <p class="project-summary">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="tech-stack">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <button class="btn-primary-outline">See project</button>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="image">
+          <img src="images/work4.png" alt="fourth project" />
+        </div>
+
+        <div class="card-content">
+          <h3 class="card-title"></h3>
+          <ul class="project-experience">
+            <li class="first-list">CANOPY</li>
+            <li class="other-list"><span></span>Full Stack Dev</li>
+            <li class="other-list"><span></span>2015</li>
+          </ul>
+          <p class="project-summary">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="tech-stack">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li>
+          </ul>
+          <button class="btn-primary-outline">See project</button>
+        </div>
+      </div>
+    </section>
+    <!-- </div> -->
+    <!-- About -->
+    <section class="about">
+      <div class="about-container">
+        <h2 class="about-title">About Myself</h2>
```

Comment:
- [ ] Please I highly suggest increasing the pixels `px` to the "About Myself" to fit the required [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=37-2238&t=HQA60gDOdof1M29t-0). 😄 

_**REQUIRED FIGMA DESIGN**_

![Screenshot (252)](https://user-images.githubusercontent.com/78104544/235191911-a3e87ef9-1005-4579-bc92-5be827b0efec.png)

_**YOUR DESIGN**_


![Screenshot (250)](https://user-images.githubusercontent.com/78104544/235192105-c8c5e625-271c-409d-a7b9-3902ce696667.png)

Reply:
increased the px

Reply:
@braincee thanks for the review,